apps/prairielearn/src/pages/instructorAssessmentOverrides/instructorAssessmentOverrides.sql

```diff
@@ -0,0 +1,208 @@
+-- BLOCK select_assessment_access_policy
+SELECT
+  COALESCE(
+    format_date_full_compact (aap.created_at, 'America/Chicago'),
+    '—'
+  ) AS created_at,
+  COALESCE(aap.created_by::text, '—') AS created_by,
+  COALESCE(aap.credit::text, '—') AS credit,
+  COALESCE(
+    format_date_full_compact (aap.end_date, 'America/Chicago'),
+    '—'
+  ) AS end_date,
+  COALESCE(aap.note, '—') AS note,
+  COALESCE(
+    format_date_full_compact (aap.start_date, 'America/Chicago'),
+    '—'
+  ) AS start_date,
+  COALESCE(
+    (
+      SELECT
+        name
+      from
+        groups
+      where
+        id = aap.group_id
+    ),
+    '-'
+  ) as group_name,
+  COALESCE(
+    (
+      SELECT
+        uid
+      FROM
+        users
+      WHERE
+        user_id = aap.user_id
+    ),
+    '-'
+  ) AS student_uid
+FROM
+  assessment_access_policies AS aap
+WHERE
+  assessment_id = $assessment_id
+ORDER BY
+  aap.created_at;
+
+-- BLOCK select_group_in_assessment
+SELECT
+  groups.id,
+  groups.name,
+  groups.group_config_id
+FROM
+  groups
+  JOIN group_configs ON groups.group_config_id = group_configs.id
+WHERE
+  groups.course_instance_id = $course_instance_id
+  AND groups.name = $group_name
+  AND group_configs.assessment_id = $assessment_id;
+
+-- BLOCK insert_assessment_access_policy
+INSERT INTO
+  assessment_access_policies (
+    assessment_id,
+    created_at,
```

Comment:
This column has a default of `NOW()`, there's no need to explicitly provide a default.